addons/vehicleTrenches/functions/fnc_lowerPlow.sqf

```diff
@@ -0,0 +1,56 @@
+#include "script_component.hpp"
+/*
+ * Author: Ashes
+ * Lower the plow for the target vehicle, setting the plow to the specified mode.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Plow Mode <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_vehicle, PLOW_NONE] call sws_vehicletrenches_fnc_lowerPlow
+ *
+ * Public: No
+ */
+
+params ["_vehicle", "_plowMode"];
+
+// Grab config for the vehicle's plow
+private _config = configFile >> "CfgVehicles" >> typeOf _vehicle >> "PlowConfig";
+if (!(isClass _config)) exitWith {
```

Comment:
either remove this check or make it `if !(isClass _config) exitWith {ERROR_1("Attempted to lowerPlow on invalid vehicle",_vehicle)};`

Reply:
Generally diag_log isn't to be used directly, prefer the cba macros for debug info